data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/v2/debug/GetStateIntegrationTest.java

```diff
@@ -25,7 +25,7 @@
 import org.junit.jupiter.api.Test;
 import tech.pegasys.teku.api.schema.Version;
 import tech.pegasys.teku.beaconrestapi.AbstractDataBackedRestAPIIntegrationTest;
-import tech.pegasys.teku.beaconrestapi.handlers.v2.debug.GetState;
+import tech.pegasys.teku.beaconrestapi.handlers.v1.beacon.GetState;
```

Comment:
my expectation here would be the actual test moves location, not just the GetSTate it calls.

Reply:
Yes. Haven't gotten to cleaning up the testing yet, so that will be cleaned up